### PR TITLE
Api

### PR DESCRIPTION
**Pull request by Salwa99**
# connect to API
### Howdy :cowboy_hat_face: 

**In this PR I:**

1. Use fetch  with createAsyncThunk for making API calls.
2. Fetch book list from Bookstore API on Books page load.
3. Refactor the add book and remove book features .
4. Fix Linters errors.
5. Follow GitFlow

_Cheers_

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,43 +1,46 @@
...
-    default:
-      return state;
+    case (LOAD):
+      return action.state;
+    default: return state;
   }
+}
+
+// Action Creators
+export const loadBooks = () => async (dispatch) => {
```
Comment (topeogunleye) [inline src/redux/books/books.js]: - [x] As required by this project, please make use of [`createAsyncThunk`](https://redux-toolkit.js.org/api/createAsyncThunk) from `@redux/toolkit` to handle redux `thunks` as this is the recommended way by redux's developers

// The below code is just an example and will need to be adapted for your project.
```

import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
import axios from 'axios'; 


export const getBooks = createAsyncThunk(
  'books/getBooks',
  async () => {
    const response = await axios.get(
      'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/',
    );
    return response.data;
  },
); 

```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Salwa99, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (Salwa99) [reply src/redux/books/books.js]: Thank you @topeogunleye , I'll fix it

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (topeogunleye) [reply src/redux/books/books.js]: @Salwa99, you're welcome 😄

**Pull request merged**